src/oscar/apps/dashboard/catalogue/views.py

```diff
@@ -61,7 +61,7 @@ def filter_products(queryset, user):
     return queryset.filter(stockrecords__partner__users__pk=user.pk).distinct()
 
 
-class ProductListView(SingleTableMixin, generic.TemplateView):
+class ProductListView(SingleTableMixin, generic.ListView):
```

Comment:
Looking through the source, you are right :-)

Perhaps better to extend from SingleTableView, see https://github.com/bradleyayers/django-tables2/blob/37a64ab92bdfb04f56eebf7fd01fc245ae568301/django_tables2/views.py#L112